Recipe_1.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Pizza</title>
+</head>
+<body>
+  <header>
+    <nav>
+     <a href="index.html" class="logo">Happy<span1>Kitchen</span></a>
+    </nav>
+    <ul class="navbar">
+      <li><a href="Recipe_1.html" class="nav-list">Pi<span1>zza</span1> |</a></li>
+      <li><a href="Recipe_2.html" class="nav-list"><span1>Bur</span1>ger |</a></li>
+      <li><a href="Recipe_3.html" class="nav-list">Sa<span1>lad</span1></a></li>
+    </ul>
+  </header>
+  <img src="/images/image-for-pizza.jpg" alt="pizza image" class="pizza-1">
+  <div class="ingredients">
+    <h4 class="Pizza-dough">Pizza Dough Ingredients</h4>
+    <p class="pizza-p">Makes enough for two 10-inch to 12-inch (25.4-cm to 30.5-cm) pizzas</p>
+    <nav>
+      <ul class="list-ingredient">
+        <li>1-1/2 cups (375 ml) warm water</li>
+        <li>1 package or 2-1/4 tsp (11.25 ml) active dry yeast</li>
+        <li>3-1/2 cups (875 ml) bread flour</li>
+        <li>2 Tbsp (30 ml) olive oil</li>
+        <li>2 tsp (10 ml) salt</li>
+        <li>1 tsp (5 ml) sugar</li>
+      </ul>
+    </nav>
+  </div>
```

Comment:
Following standard practice of html semantics, <code>nav</code> can only be a parent to navigation links. Kindly remove the nav tag from this list.